virt.te

```diff
@@ -1674,6 +1771,213 @@ corenet_rw_tun_tap_dev(virt_bridgehelper_t)
+manage_files_pattern(virtinterfaced_t, virtinterfaced_var_run_t, virtinterfaced_var_run_t)
+files_pid_filetrans(virtinterfaced_t, virtinterfaced_var_run_t, { dir file } )
+filetrans_pattern(virtinterfaced_t, virt_var_run_t, virtinterfaced_var_run_t, { dir file } )
+
+read_files_pattern(virtinterfaced_t, virt_etc_t, virt_etc_t)
+
+#######################################
+#
+# virtnetworkd local policy
+#
+allow virtnetworkd_t self:capability { kill sys_ptrace };
+allow virtnetworkd_t self:netlink_netfilter_socket create_socket_perms;
+allow virtnetworkd_t self:process setcap;
+
+allow virtnetworkd_t virt_var_run_t:dir { search_dir_perms };
+manage_dirs_pattern(virtnetworkd_t, virtnetworkd_var_run_t, virtnetworkd_var_run_t)
+manage_files_pattern(virtnetworkd_t, virtnetworkd_var_run_t, virtnetworkd_var_run_t)
+files_pid_filetrans(virtnetworkd_t, virtnetworkd_var_run_t, { dir file } )
+filetrans_pattern(virtnetworkd_t, virt_var_run_t, virtnetworkd_var_run_t, { dir file } )
+
+read_files_pattern(virtnetworkd_t, virt_etc_t, virt_etc_t)
+manage_dirs_pattern(virtnetworkd_t, virt_etc_rw_t, virt_etc_rw_t)
+manage_files_pattern(virtnetworkd_t, virt_etc_rw_t, virt_etc_rw_t)
+read_lnk_files_pattern(virtnetworkd_t, virt_etc_rw_t, virt_etc_rw_t)
+
+manage_dirs_pattern(virtnetworkd_t, virt_var_lib_t, virt_var_lib_t)
+manage_files_pattern(virtnetworkd_t, virt_var_lib_t, virt_var_lib_t)
+
+kernel_read_network_state(virtnetworkd_t)
+
+sysnet_read_config(virtnetworkd_t)
+
+optional_policy(`
+        dnsmasq_domtrans(virtnetworkd_t)
+        dnsmasq_manage_pid_files(virtnetworkd_t)
+')
+
+optional_policy(`
+        iptables_domtrans(virtnetworkd_t)
+        iptables_read_var_run(virtnetworkd_t)
+')
+
+#######################################
+#
+# virtnodedevd local policy
+#
+allow virtnodedevd_t self:capability sys_admin;
+allow virtnodedevd_t self:netlink_generic_socket create_socket_perms;
+
+allow virtnodedevd_t virt_var_run_t:dir { create search_dir_perms };
+manage_dirs_pattern(virtnodedevd_t, virtnodedevd_var_run_t, virtnodedevd_var_run_t)
+manage_files_pattern(virtnodedevd_t, virtnodedevd_var_run_t, virtnodedevd_var_run_t)
+files_pid_filetrans(virtnodedevd_t, virtnodedevd_var_run_t, { dir file } )
+filetrans_pattern(virtnodedevd_t, virt_var_run_t, virtnodedevd_var_run_t, { dir file } )
+
+read_files_pattern(virtnodedevd_t, virt_etc_t, virt_etc_t)
+
+dev_rw_mtrr(virtnodedevd_t)
+
+miscfiles_read_hwdata(virtnodedevd_t)
+
+optional_policy(`
+        udev_read_pid_files(virtnodedevd_t)
+')
+
+#######################################
+#
+# virtnwfilterd local policy
+#
+allow virtnwfilterd_t self:capability net_raw;
+allow virtnwfilterd_t self:netlink_netfilter_socket create_socket_perms;
+allow virtnwfilterd_t self:rawip_socket create_socket_perms;
+
+allow virtnwfilterd_t virt_var_run_t:dir { create search_dir_perms };
+manage_dirs_pattern(virtnwfilterd_t, virtnwfilterd_var_run_t, virtnwfilterd_var_run_t)
```

Comment:
These set of rules could be part of `virt_driver_template`

Reply:
ok, thanks